shopfloor/services/validator.py

```diff
@@ -1,6 +1,70 @@
-# Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+# Copyright 2020-2021 Camptocamp SA (http://www.camptocamp.com)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from odoo.addons.component.core import AbstractComponent
+import logging
+
+from odoo.addons.component.core import AbstractComponent, Component
+from odoo.addons.component.exception import NoComponentError
+
+_logger = logging.getLogger(__name__)
+
+
+class ShopfloorRestCerberusValidator(Component):
```

Comment:
can you report here in the docstring the reason behind this as done in the comment in the patch removed?